packages/amesos2/src/Amesos2_Superludist_def.hpp

```diff
@@ -1037,9 +1037,8 @@ namespace Amesos2 {
       Teuchos::RCP<const MatrixAdapter<Matrix> > redist_mat
         = this->matrixA_->get(ptrInArg(*superlu_rowmap_));
 
-      // ssame as A's
-      superlu_contig_rowmap_ = this->matrixA_->getRowMap();
-      superlu_contig_colmap_ = this->matrixA_->getColMap();
+      // ssame as A's target
+      superlu_contig_rowmap_ = superlu_rowmap_;
```

Comment:
Looks like this is the `is_contiguous` else branch, so reusing the existing rowmap?

Reply:
Yes, this is just the original code pass, and the `rowmap` is created in initialization to reorder GIDs in the ascending order, I think.